addons/website_sale/static/src/js/product/product.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="sale.Product" t-inherit-mode="extension">
+        <xpath
+            expr="//div[@name='sale_product_configurator_quantity_buttons']"
+            position="attributes">
```

Comment:
nitpicking: I know it doesn't feel natural but I would prefer applying our convention 100% or keeping all on one line
```suggestion
        <xpath
            expr="//div[@name='sale_product_configurator_quantity_buttons']"
            position="attributes"
        >
```

Reply:
Done here and elsewhere (just note that none of the product configurator components in `sale` follow this convention, so I didn't adjust those).